sycl/doc/extensions/supported/sycl_ext_intel_fpga_lsu.md

```diff
@@ -126,113 +126,119 @@ this extension may change these APIs in ways that are incompatible with the
 versions described here.
 
 In the experimental API version, member functions `load()` and `store()` take
-template arguments, which can contain the latency control properties
-`latency_anchor_id` and/or `latency_constraint`.
+in a property list as function argument, which can contain the latency control
+properties `latency_anchor_id` and/or `latency_constraint`.
 
-1. **`sycl::ext::intel::experimental::latency_anchor_id<N>`, where `N` is an integer**:
+1. **`sycl::ext::oneapi::experimental::latency_anchor_id<N>`, where `N` is an integer**:
 represents ID of the current function call when it performs as an anchor. The ID
 must be unique within the application, with a diagnostic required if that
 condition is not met.
-2. **`sycl::ext::intel::experimental::latency_constraint<A, B, C>`** contains control
+2. **`sycl::ext::oneapi::experimental::latency_constraint<A, B, C>`** contains control
 parameters when the current function performs as a non-anchor, where:
     - **`A` is an integer**: The ID of the target anchor defined on a different
     instruction through a `latency_anchor_id` property.
     - **`B` is an enum value**: The type of control from the set
-    {`type::exact`, `type::max`, `type::min`}.
+    {`latency_control_type::exact`, `latency_control_type::max`, `latency_control_type::min`}.
     - **`C` is an integer**: The relative clock cycle difference between the
     target anchor and the current function call, that the constraint should
     infer subject to the type of the control (exact, max, min).
 
-The template arguments above don't have to be specified if user doesn't want to
-apply latency controls. The template arguments can be passed in arbitrary order.
-
 ### Implementation
```

Comment:
Is there a reason to have so many implementation details here, rather than a simple synopsis as you have in the other spec?  All of the comments I made above in the other spec's synopsis also apply here.

Reply:
It's just because we put full non-experimental implementation in this doc before.
I prefer to have just synopsis here too lol. If it's acceptable, I would be glad to document just synopsis here too.